### PR TITLE
Osdocs 9783 take4

### PR DESCRIPTION
**Pull request by jmanthei**
Version(s):

Issue: https://issues.redhat.com/browse/OSDOCS-9783 

Link to docs preview: https://76321--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -245,7 +245,7 @@ For more information, see xref:../edge_computing/cnf-talm-for-cluster-upgrades.a
 [id="ocp-4-16-sha-haproxy-support-removed"]
 === SHA1 certificates no longer supported for use with HAProxy
```
Comment (gcs278) [inline release_notes/ocp-4-16-release-notes.adoc:246]: Supposedly, SHA-1 is the proper spelling for SHA1. I didn't know this until recently. For example, https://en.wikipedia.org/wiki/SHA-1. 
 
```suggestion
=== SHA-1 certificates no longer supported for use with HAProxy
```

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -245,7 +245,7 @@ For more information, see xref:../edge_computing/cnf-talm-for-cluster-upgrades.a
...
 
-SHA1 certificates are no longer supported for use with HaProxy. Both existing and new routes using SHA1 certificates in {product-version} are rejected and no longer function.
+SHA1 certificates are no longer supported for use with HAProxy. Both existing and new routes using SHA1 certificates in 4.16 are rejected and no longer function. For more information about creating secure routes, see xref:../networking/routes/secured-routes.html[Secured Routes].
```
Comment (gcs278) [inline release_notes/ocp-4-16-release-notes.adoc:248]:
```suggestion
SHA-1 certificates are no longer supported for use with HAProxy. Both existing and new routes using SHA1 certificates in 4.16 are rejected and no longer function. For more information about creating secure routes, see xref:../networking/routes/secured-routes.html[Secured Routes].
```

Comment (jmanthei) [reply release_notes/ocp-4-16-release-notes.adoc:248]: ty

File: release_notes/ocp-4-16-release-notes.adoc (hunk 1 above)
Comment (jmanthei) [reply release_notes/ocp-4-16-release-notes.adoc:246]: ty

File: release_notes/ocp-4-16-release-notes.adoc [hunk 3]
```diff
@@ -243,9 +243,9 @@ For more information, see xref:../edge_computing/cnf-talm-for-cluster-upgrades.a
 
 [discrete]
 [id="ocp-4-16-sha-haproxy-support-removed"]
-=== SHA1 certificates no longer supported for use with HAProxy
+=== SHA-1 certificates no longer supported for use with HAProxy
 
-SHA1 certificates are no longer supported for use with HAProxy. Both existing and new routes using SHA1 certificates in 4.16 are rejected and no longer function. For more information about creating secure routes, see xref:../networking/routes/secured-routes.html[Secured Routes].
+SHA-1 certificates are no longer supported for use with HAProxy. Both existing and new routes using SHA1 certificates in 4.16 are rejected and no longer function. For more information about creating secure routes, see xref:../networking/routes/secured-routes.html[Secured Routes].
```
[the remark this answers is not in the document]
Comment (jmanthei) [reply release_notes/ocp-4-16-release-notes.adoc:248]: oh my, I should have caught that one!

**Comment by jmanthei** [comment]
/label peer-review-needed

Requesting merge review immediately following as we are only updating SHA1 to SHA-1 for the release note.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 4]
```diff
@@ -243,9 +243,9 @@ For more information, see xref:../edge_computing/cnf-talm-for-cluster-upgrades.a
 
 [discrete]
 [id="ocp-4-16-sha-haproxy-support-removed"]
-=== SHA1 certificates no longer supported for use with HAProxy
+=== SHA-1 certificates no longer supported for use with HAProxy
 
-SHA1 certificates are no longer supported for use with HaProxy. Both existing and new routes using SHA1 certificates in {product-version} are rejected and no longer function.
+SHA-1 certificates are no longer supported for use with HAProxy. Both existing and new routes using SHA-1 certificates in 4.16 are rejected and no longer function. For more information about creating secure routes, see xref:../networking/routes/secured-routes.html[Secured Routes].
```
Comment (mburke5678) [inline release_notes/ocp-4-16-release-notes.adoc:248]: I might leave the {product-title} in there to be clear. I like the addition of the version number.
```suggestion
SHA-1 certificates are no longer supported for use with HAProxy. Both existing and new routes using SHA-1 certificates in {product-title} 4.16 are rejected and no longer function. For more information about creating secure routes, see xref:../networking/routes/secured-routes.html[Secured Routes].
```

Comment (jmanthei) [reply release_notes/ocp-4-16-release-notes.adoc:248]: Sure thing!

**Pull request merged**